js/libs/form.js

```diff
@@ -133,6 +133,16 @@ Fliplet.Widget.instance('form-builder', function(data) {
 
           var showCurrentDateTime = field.autofill === 'always';
 
+          if (field._type === 'flCheckbox') {
```

Comment:
why don't you move this to the switch case?

Reply:
The `fieldData` typecasting needs to happen before we go into the switch cases.